### PR TITLE
licensing doc tweaks

### LINKED ISSUE #64: license & contributing
There are two distinct sections for Contributing & License in the README, one pointing to "Open-2 Contributing" (an out of date contributing template that I made some time ago) and an ISC License, the other to Esri/contributing and Apache 2.0. It would be good for users to clarify licensing and contributing and make sure that information is accurately reflected in the README and package.json.

### PR DESCRIPTION
**Pull request by jgravois**
attempt to resolve #64.  happy to incorporate more feedback.

### CODE REVIEW

**Comment by alaframboise** [comment]
We talked about adding a "Documentation" header with a link to the doc as well.

**Comment by jgravois** [comment]
> We talked about adding a "Documentation" header

added.

File: README.md [hunk 1]
```diff
@@ -10,27 +9,17 @@
...
 Node/Browser Javascript library for ArcGIS
 
-## Install
-
 ```
 npm install arcgis
 ```
 
-## Usage
-
```
Comment (ungoldman) [inline README.md]: Drive-by comments from a quitter:

1. I think having `Install` and `Usage` sections up front is a good pattern worth keeping (and expanding).
2. https://github.com/Esri/node-arcgis/blob/lic/LICENSE.md and https://github.com/Esri/node-arcgis/blob/lic/license.txt conflict, you probably want to delete LICENSE.md.

Comment (jgravois) [inline README.md]: agreed on all points.

@nikolaswise already has a jump start on this in #68. so if he wants to rip off my fixes for the `package.json` license syntax and website url update, we'll just close this one.
